src/utils/removeDateEntity.m

```diff
@@ -0,0 +1,41 @@
+function removeDateEntity(varargin)
+    %
+    % Removes the date entity from all the files in a BIDS data set
+    %
+    %
+    % USAGE::
+    %
+    %   removeDateEntity(pathToDataSet, 'filter', filter)
+    %
+    %
+    % (C) Copyright 2022 CPP_BIDS developers
```

Comment:
There is now a stand alone function to remove date entities

Reply:
cool! I think I'm doing something wrong with the `pathToDataSet` variable.